tables/fr-bfu-comp6.utb

```diff
@@ -265,19 +265,46 @@ numsign 6             # number sign, just a dots operand
 numericmodechars ^,:.+-*/()=[]
 numericnocontchars 0123456789
 
+# Capitalization
 capsletter 46
 begcapsword 46-46
 endcapsword 6-3 # The first method is not compatible with liblouis, so use of the alternative method (official but but less common)
+capsmodechars -'./
+# Capital 'phrase' not possible: https://github.com/liblouis/liblouis/issues/706
+# begcaps 25-46
+# endcaps 46
 
+# Emphasis
 emphclass italic
 emphclass underline
 emphclass bold
-begemphphrase italic 456-456
-endemphphrase italic before 456
-begemphword italic 456
 emphletter italic 456
-begemphphrase bold 456-456
+emphletter underline 456
+emphletter bold 456
+begemphword italic 456
+begemphword underline 456
+begemphword bold 456
+endemphword italic 6-3
+endemphword underline 6-3
+endemphword bold 6-3
+begemphphrase italic 25-456
+endemphphrase italic before 456
+lenemphphrase italic 4
+begemphphrase underline 25-456
+endemphphrase underline before 456
+lenemphphrase underline 4
+begemphphrase bold 25-456
 endemphphrase bold before 456
+lenemphphrase bold 4
+
+# ToDo (don't pass YAML tests)
```

Comment:
Do you know if the tests need to be adapted or if these rules don't do what you intend?

Reply:
The tests must be refine with ambiguous cases, especially for backward translation, to have robust base.
After that, I need to dig in the doc and understand how opcodes/multipass/etc. works precisely.
At this time, the rules 'noback correct' do the job but there's a conflict with other rules.